index.js

```diff
@@ -115,11 +115,12 @@ const openArchive = function (file, noCache) {
 // ensure non-breaking changes
 exports.get = (file, archive) => {
   log.debug('get() : ' + file + ', ' + archive);
-  if (['darwin', 'linux'].indexOf(process.platform) > -1) {
-    return exports.extractFile(file, archive)
-  } else {
-    return exports.extractFileJS(file, archive)
-  }
+  return exports.extractFileJS(file, archive);
```

Comment:
Why did this get removed?

Reply:
hmmm think that was a change from a different branch that leaked into this one, will fix

Reply:
ok fixed and also I remembered why that happened.
I haven't verified, but I got a few reports from a linux user saying that the extraction sometimes failed on that platform but was fine with the JS only version. Performance wasn't a concern, so we disabled the platform check in a separate branch, which leaked over here.